index.html

```diff
@@ -56,5 +56,141 @@
             </div>
         </section>
     </header>
+    <section class="works" id="portfolio">
+        <div class="work-1 work--1">
+            <h1>My Recent Works</h1>
+            <img src="img/Vector-line.png" alt="line" class="line">
+        </div>
+        <div class="container-work">
+            <div class="work--2"></div>
+            <div class="work--3">
+                <h1>Multi-Post Stories </h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="buttons">
+                    <li class="button"><a href="">css</a></li>
+                    <li class="button"><a href="">html</a></li>
+                    <li class="button"><a href="">bootstrap</a></li>
+                    <li class="button"><a href="">Ruby</a></li>
+                </ul>
+                <button class="button2 dark-font" type="submit">Project</button>
+            </div>
+        </div>
+        <div class="grid-container" id="grid-container">
+            <div class="work work--4-1">
+                <div class="container">
+                    <button type="button" class="link" id="myBtn">See Project</button>
```

Comment:
- [ ] I think you should consider styling the buttons to match the one in the [Figma design template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) 
. Kindly pay attention to the font color, background, and border 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/176787099-d51e8ad5-b6c4-4ac1-be6d-69bba80680df.png)  |  ![image](https://user-images.githubusercontent.com/63473584/176787165-1160c01e-6301-47c9-8bf9-a794a77d6583.png)  |